src/main/java/com/example/todo/TaskManager.java

```diff
@@ -27,7 +27,15 @@ public void removeTask(int id) {
     }
 
     public void markTaskAsCompleted(int id) {
-        // TODO: Implement this method
+        for (Task task : tasks) {
+            if (task.getId() == id) {
+                task.setCompleted(true);
+                System.out.println("Task with ID " + id + " marked as completed.");
+                return;
+            }
+        }
+        
+        System.out.println("Task with ID " + id + " not found.");
```

Comment:
```suggestion
        System.out.println("Task with ID " + id + " not (test change asdasd) found.");
```
Suggestion... (Sent from code spaces)